src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -466,17 +466,24 @@ private function createBlock(iterable $messages, string $type = null, string $st
         }
 
         // wrap and add newlines for each element
+        $messagesLines = [];
```

Comment:
why this variable? can't we append to `$lines` directly?

Reply:
Yes, good point.